src/Node.php

```diff
@@ -430,4 +434,137 @@ public function getDocCommentText() {
+            if ($useDeclaration->groupClauses !== null) {
+                // use A\B{C, D} => multiple imports
+                $imports = \array_filter($useDeclaration->groupClauses->children, function($value) {
+                    return $value instanceof NamespaceUseGroupClause;
+                });
+            } else {
+                // use A\B\C; => single import
+                $imports = [$useDeclaration];
+            }
+
+            $namespaceNamePartsPrefix =
+                $useDeclaration->namespaceName !== null ? $useDeclaration->namespaceName->nameParts->children : [];
+
+            foreach ($imports as $import) {
+                if ($useDeclaration->groupClauses !== null) {
+                    // use A\B\C\{D\E};                 namespace import: ["E" => [A,B,C,D,E]]
+                    // use A\B\C\{D\E as F};            namespace import: ["F" => [A,B,C,D,E]]
+                    // use function A\B\C\{A, B}        function import: ["A" => [A,B,C,A], "B" => [A,B,C]]
+                    // use function A\B\C\{const A}     const import: ["A" => [A,B,C,A]]
+                    $alias = $import->namespaceName->getLastNamePart()->getText($contents);
+                    $namespaceNameParts = \array_merge($namespaceNamePartsPrefix, $import->namespaceName->nameParts->children);
+                    $functionOrConst = $import->functionOrConst ?? $useDeclaration->functionOrConst;
+                } else {
+                    // use A\B\C;               namespace import: ["C" => [A,B,C]]
+                    // use A\B\C as D;          namespace import: ["D" => [A,B,C]]
+                    // use function A\B\C as D  function import: ["D" => [A,B,C]]
+                    // use A\B, C\D;            namespace import: ["B" => [A,B], "D" => [C,D]]
+                    $alias = $useDeclaration->namespaceAliasingClause === null
+                        ? $useDeclaration->namespaceName->getLastNamePart()->getText($contents)
+                        : $useDeclaration->namespaceAliasingClause->name->getText($contents);
+
+                    $namespaceNameParts = $namespaceNamePartsPrefix;
+                    $functionOrConst = $useDeclaration->functionOrConst;
+                }
+
+                // Add the alias and resolved name to the corresponding namespace, function, or const import table.
+                // If the alias already exists, it will get replaced by the most recent using.
+                // TODO - worth throwing an error here instead?
+                if ($alias !== null) {
+                    if ($functionOrConst === null) {
+                        // namespaces are case-insensitive
+                        $alias = \strtolower($alias);
+                        $namespaceImportTable[$alias] = ResolvedName::buildName($namespaceNameParts, $contents);
+                    } elseif ($functionOrConst->kind === TokenKind::FunctionKeyword) {
+                        // functions are case-insensitive
```

Comment:
Yep, that's a `wontfix` for the LS. See https://github.com/felixfbecker/php-language-server/issues/69 for context